iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractTapChanger.java

```diff
@@ -145,13 +145,12 @@ public double getTargetDeadband() {
     }
 
     public C setTargetDeadband(double targetDeadband) {
-        if (!Double.isNaN(targetDeadband) && targetDeadband < 0) {
-            throw new ValidationException(parent, "Unexpected value for target deadband of phase tap changer: " + targetDeadband);
-        }
+        double realTargetDeadband = Double.isNaN(targetDeadband) ? 0 : targetDeadband;
```

Comment:
As the target deadband is required only if the shunt is regulating (regarding to the javadoc you wrote), why don't you accept NaN? Maybe the `checkTargetDeadband` method should take two parameters?

If you set the targetDeadband to NaN and call the getTargetDeadband then, you will have 0 ? It's strange to me...